src/main/java/tsunami/security/scanner/utilities/App.java

```diff
@@ -43,36 +49,46 @@ public static void main(String[] args) throws IOException, ApiException, ClassNo
     System.out.println(
         "App: " + appName + " Config Path: " + appConfigPath + " templateData: " + templateData);
 
+    // Set a default directory for configs if no config path is passed in.
+    if (isNullOrEmpty(appConfigPath)) {
+      appConfigPath = System.getProperty("user.dir") + "/application";
+    }
+
     // Combine the file path with application name as a directory.
     String configPath = appConfigPath + "/" + appName + "/";
 
     // Transform input template data Json String to Map.
     Map<String, String> templateDataMap = TemplateDataUtil.parseTemplateDataJson(templateData);
 
-    // Initialize Kubernetes Java Client Api.
-    ApiClient client = Config.defaultClient();
-    Configuration.setDefaultApiClient(client);
-
     // Load all application's config files, run services and deploy the app on GKE.
     try {
       File configFiles = new File(configPath);
 
+      // Check if configFiles is an existed directory
+      if (!configFiles.isDirectory()) throw new FileNotFoundException("Wrong directory.");
+
       // Traverse all files under certain application's config path
       for (File configFile : Files.fileTraverser().depthFirstPreOrder(configFiles)) {
 
         if (configFile.isFile()) {
           String resourceConfig = FreeMarkerUtil.replaceTemplates(templateDataMap, configFile);
 
           // Parse all config to Kubernetes Objects and create them.
-          KubeJavaClientUtil.createResources(resourceConfig);
+          kubeJavaClientUtil.createResources(resourceConfig);
         }
       }
     } catch (Exception e) {
       e.printStackTrace();
+      throw e;
     }
```

Comment:
Instead of catch and rethrow the same exception, just remove the entire `try {} catch {}` block and add expected exceptions to the `throws XXXException` part in the method definition.